bridge/engine/ActiveMediaList.cpp

```diff
@@ -391,7 +391,7 @@ void ActiveMediaList::process(const uint64_t timestampMs, bool& outDominantSpeak
     {
         const auto& top = heap.top();
         updateActiveAudioList(top.participant);
-        if (_c9_conference && top.score - top.noiseLevel > _activeTalkerSilenceThresholdDb)
+        if (!_c9_conference && top.score - top.noiseLevel > _activeTalkerSilenceThresholdDb)
```

Comment:
Are we not reporting a fake audio level when ptt=1? 
Then why do we build activeTalkersSnapshot in two places. This condition should suffice for both c9 and !c9. 
Instead we remove the adding to activeTalkersSnapshot in rankSpeakers.

Reply:
Fixed.